widgetssdk/src/main/java/com/glia/widgets/view/unifiedui/config/chat/Chat.kt

```diff
@@ -0,0 +1,50 @@
+package com.glia.widgets.view.unifiedui.config.chat
+
+import android.os.Parcelable
+import com.glia.widgets.view.unifiedui.config.base.ColorLayer
+import com.glia.widgets.view.unifiedui.config.base.Header
+import com.glia.widgets.view.unifiedui.config.base.Layer
+import com.google.gson.annotations.SerializedName
+import kotlinx.parcelize.Parcelize
+
+@Parcelize
+data class Chat(
```

Comment:
The same idea here: I'm not sure, but an idea came to my mind that it would make the code clear if we named all style components like this: `ChatStyle` or `ChatScreenStyle`. Otherwise, it may be confusing: what kind of `Chat`?